fhir-persistence-cos/pom.xml

```diff
@@ -0,0 +1,112 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>fhir-persistence-cos</artifactId>
```

Comment:
module overview should be added to the README:  https://github.com/IBM/FHIR/blob/main/README.md#server